guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -36,21 +36,93 @@ ifdef::foreman-el,foreman-deb[]
+--foreman-initial-organization "_<My_Organization>_" \
+--foreman-initial-location "_<My_Location>_" \
+--foreman-initial-admin-username _<admin_user_name>_ \
+--foreman-initial-admin-password _<admin_password>_
 ----
+ifdef::katello,satellite,orcharhino[]
+** To install {ProjectServer} with an external PostgreSQL server, enter the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-initial-organization "_<My_Organization>_" \
+--foreman-initial-location "_<My_Location>_" \
+--foreman-initial-admin-username _<admin_user_name>_ \
+--foreman-initial-admin-password _<admin_password>_ \
+--katello-candlepin-manage-db false \
+--katello-candlepin-db-host _<postgres.example.com>_ \
+--katello-candlepin-db-name _<candlepin>_ \
+--katello-candlepin-db-user _<candlepin>_ \
+--katello-candlepin-db-password _<Candlepin_Password>_ \
+--foreman-proxy-content-pulpcore-manage-postgresql false \
+--foreman-proxy-content-pulpcore-postgresql-host _<postgres.example.com>_ \
+--foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
+--foreman-proxy-content-pulpcore-postgresql-user pulp \
+--foreman-proxy-content-pulpcore-postgresql-password _<Pulpcore_Password>_ \
+--foreman-db-manage false \
+--foreman-db-host _<postgres.example.com>_ \
+--foreman-db-database _<foreman>_ \
+--foreman-db-username _<foreman>_ \
+--foreman-db-password _<Foreman_Password>*_
+----
++
+To also enable encrypted connections for these external databases, use the following command instead:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-initial-organization "_<My_Organization>_" \
+--foreman-initial-location "_<My_Location>_" \
+--foreman-initial-admin-username _<admin_user_name>_ \
+--foreman-initial-admin-password _<admin_password>_ \
+--katello-candlepin-manage-db false \
+--katello-candlepin-db-host _<postgres.example.com>_ \
+--katello-candlepin-db-name _<candlepin>_ \
+--katello-candlepin-db-user _<candlepin>_ \
+--katello-candlepin-db-password _<Candlepin_Password>_ \
+--foreman-proxy-content-pulpcore-manage-postgresql false \
+--foreman-proxy-content-pulpcore-postgresql-host _<postgres.example.com>_ \
+--foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
+--foreman-proxy-content-pulpcore-postgresql-user pulp \
+--foreman-proxy-content-pulpcore-postgresql-password _<Pulpcore_Password>_ \
+--foreman-db-manage false \
+--foreman-db-host _<postgres.example.com>_ \
+--foreman-db-database _<foreman>_ \
+--foreman-db-username _<foreman>_ \
+--foreman-db-password _<Foreman_Password>*_
+--foreman-db-root-cert _<My_CA_Certificate>_ \
+--foreman-db-sslmode verify-full \
+--foreman-proxy-content-pulpcore-postgresql-ssl true \
+--foreman-proxy-content-pulpcore-postgresql-ssl-root-ca _<My_CA_Certificate>_ \
+--katello-candlepin-db-ssl true \
+--katello-candlepin-db-ssl-ca _<My_CA_Certificate>_ \
+--katello-candlepin-db-ssl-verify true \
```

Comment:
Please order these together with the other options so we group each connection. I've included them in the above suggestion
```suggestion
```